easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,32 @@
+# easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org
+#
+# Fred Hutchenson Cancer Research Center Seattle WA
+name = 'OpenJPEG'
+version = '2.3.0'
+
+easyblock = 'CMakeMake'
+
+homepage = 'http://www.openjpeg.org/'
+description = """OpenJPEG is an open-source JPEG 2000 codec written in
+ C language. It has been developed in order to promote the use of JPEG 2000,
+ a still-image compression standard from the Joint Photographic Experts Group
+ (JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as
+ a JPEG 2000 Reference Software."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/uclouvain/openjpeg/archive']
+sources = ['v%(version)s.zip']
+checksums = ['e0bd4ee9ab6e4f3a581fca5e01fbfb45a8ab9062baeddd1211741531b53fe07d']
+
+separate_build_dir = True
+
+builddependencies = [
+    ('binutils', '2.30'),
+    ('CMake', '3.11.4')
+]
+
```

Comment:
@fizwit Please include a custom `sanity_check_paths`, to check for specific binaries in `bin/` and libraries in `lib*/` (and perhaps also header files in `include`, etc.).

Just checking for a couple key files is sufficient, the check is not supposed to be exhaustive.

Reply:
@fizwit You overlooked this comment w.r.t. a custom `sanity_check_paths`?